webservices/common/models/filings.py

```diff
@@ -123,8 +124,15 @@ def document_description(self):
         )
 
     @property
+    def amended_by(self):
+        if not self.superceded and not self.most_recent and self.amendment and len(self.amendment.longest_chain) > 0:
+            index = self.amendment.longest_chain.index(self.file_number)
+            return self.amendment.longest_chain[index + 1]
+        else:
+            return self.superceded
+    @property
     def is_amended(self):
-        if self.superceded is not None:
+        if self.superceded is not None or not self.most_recent:
```

Comment:
I'm not too familiar with the db schema; do we need to say `is not None`? Or is it not superfluous?